app/controllers/orders_controller.rb

```diff
@@ -9,12 +9,17 @@ def show
 
   def create
     if current_user
-      order = Order.create(user_id: current_user.id)
-      cart.create_order_items(order)
-      session.delete(:cart)
-      TwilioNotifier.new(order).notify
-      flash[:success] = "Your order has been created."
-      redirect_to order
+      if current_user.checkout_ready?
+        order = Order.create(user_id: current_user.id)
+        cart.create_order_items(order)
+        session.delete(:cart)
+        TwilioNotifier.new(order).notify
+        flash[:success] = "Your order has been created."
+        redirect_to order
+      else
+        flash[:danger] = "Your account information is incomplete."
+        redirect_to  edit_user_path(current_user)
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.